tests/Microsoft.Identity.Test.Unit.net45/PublicApiTests/ClientCredentialWithCertTest.cs

```diff
@@ -24,6 +28,16 @@ namespace Microsoft.Identity.Test.Unit
     [DeploymentItem(@"Resources\testCert.crtfile")]
     public class ConfidentialClientWithCertTests : TestBase
     {
+        private TokenCacheHelper _tokenCacheHelper;
+
+        [TestInitialize]
+        public void TestInitialize()
```

Comment:
There's a base TestInitialize, please call it instead of TestCommon.ResetInternalStaticCaches();